site/en/docs/extensions/reference/action/index.md

```diff
@@ -68,101 +63,233 @@ context.clearRect(0, 0, 16, 16);
+`action.setPopup()` method.
 
 {% Aside %}
-The `action.onClicked` event will not be dispatched if the extension action
-has specified a popup to show on click on the current tab.
+
+The `action.onClicked` event will not be dispatched if the extension action has specified a popup to
+show on click on the current tab.
+
 {% endAside %}
 
 ## Per-tab state
 
-Extension actions can have different states for each tab. For instance, you
-could set the badge text to be different on each tab (to show tab-specific state).
-You can set the value for an individual tab using the `tabId` property in the
-various setting methods on the `action` API. For instance, to set the badge
-text on a specific tab, you would do something like the following:
+Extension actions can have different states for each tab. For instance, you could set the badge text
+to be different on each tab (to show tab-specific state). You can set the value for an individual
+tab using the `tabId` property in the various setting methods on the `action` API. For instance, to
+set the badge text on a specific tab, you would do something like the following:
 
 ```js
-const tabId = getTabId();
-const tabMessage = getTabMessage(tabId);
+function getTabId() { /* ... */}
+function getTabBadge() { /* ... */}
+
 chrome.action.setBadgeText(
-    {text: tabMessage, tabId: tabId},
-    () => { ... });
+  {
+    text: getTabBadge(tabId),
+    tabId: getTabId(),
+  },
+  () => { ... }
+);
 ```
 
-If the `tabId` property is omitted, the setting is treated as a global setting.
-Tab-specific settings take priority over any global settings.
+If the `tabId` property is omitted, the setting is treated as a global setting. Tab-specific
+settings take priority over any global settings.
 
 ## Enabled state
 
-By default, toolbar actions are enabled (clickable) on every tab. You can
-control this using the `action.enable()` and `action.disable()` methods.
-This only affects whether the popup (if any) or `action.onClicked` event
-is dispatched to your extension; it does not affect the action's presence in the
-toolbar.
+By default, toolbar actions are enabled (clickable) on every tab. You can control this using the
+`action.enable()` and `action.disable()` methods. This only affects whether the popup (if any) or
+`action.onClicked` event is dispatched to your extension; it does not affect the action's presence
+in the toolbar.
+
+## Examples
+
+The following examples show some common for actions in an extension. For a more robust demonstation
+of action capabilities, see the [Action API example][sample-example] in the
+[chrome-extension-samples][repo-samples] repository.
+
+### Show a popup
+
+It's common for extensions to display a popup when the user clicks the extension's action. To
+implement this in your own extension, declare the popup in your `manifest.json` and specify the
+content Chrome should display in the popup.
```

Comment:
"content that Chrome should display" is a little clearer because "specify the content Chrome" made me stumble

Reply:
Ack